src/spikeinterface/core/generate.py

```diff
@@ -609,6 +609,25 @@ def generate_snippets(
 
 
 ## spiketrain zone ##
+
+
+def _ensure_firing_rates(firing_rates, num_units, seed):
+    rng = np.random.default_rng(seed=seed)
+
+    if isinstance(firing_rates, tuple):
+        rng = np.random.default_rng(seed=seed)
+        lim0, lim1 = firing_rates
+        firing_rates = rng.random(num_units) * (lim1 - lim0) + lim0
```

Comment:
rng.uniform:

https://numpy.org/doc/stable/reference/random/generated/numpy.random.Generator.uniform.html

Reply:
I was thinking that random was uniform no ?
Lets use uniform it is more explicit maybe.

Reply:
Yeah, but only 0 and 1. With uniform you don't need to do the math yourself as you can specificy high and low.

Reply:
I also vote for `uniform(lim0, lim1)`